WordPress-Extra/ruleset.xml

```diff
@@ -87,6 +87,7 @@
 
 	<rule ref="WordPress.PHP.DevelopmentFunctions"/>
 	<rule ref="WordPress.PHP.DiscouragedPHPFunctions"/>
+	<rule ref="WordPress.CodeAnalysis.PerformantPHPFunctions"/>
```

Comment:
Move this up two lines for alphabetical order or have it in it's own section with documentation on what this does and the links like everywhere else.